src/controllers/userController.js

```diff
@@ -46,6 +54,14 @@ const showCurrentUser = async (req, res) => {
   }
 
   res.status(StatusCodes.OK).json({ user });
+  /*
+     #swagger.summary = 'Returns information about the requesting user based on cookie session'
+     #swagger.responses[200] = {
+				description: 'User successfully fetched.',
+				schema: { user: [{ $ref: '#/definitions/PasswordlessUser' }] }
```

Comment:
this should be a single user not an array:

```suggestion
				schema: { user: { $ref: '#/definitions/PasswordlessUser' } }
```